src/main/java/org/jenkinsci/plugins/sqlplus/script/runner/SQLPlusRunnerBuilder.java

```diff
@@ -58,22 +59,22 @@
 	private String customSQLPath;
 
 	@DataBoundConstructor
-	public SQLPlusRunnerBuilder(String credentialsId, String instance, String scriptType, String script,
+	public SQLPlusRunnerBuilder(String credentialsId, String user, String password, String instance, String scriptType, String script,
```

Comment:
In the past I had to deprecate constructors, remove the `@DataBoundConstructor`, and write a new one to support updates and avoid serialization issues. But I am not sure if that's still useful today. - https://wiki.jenkins.io/JENKINS/Hint-on-retaining-backward-compatibility.html#Hintonretainingbackwardcompatibility-Scenario:Addinganewfield